### PR TITLE
[maven-release-plugin] Install packages when building integration tests

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: dockerfiles/compilation/java-8/integration_tests.sh [hunk 1]
```diff
@@ -53,5 +53,5 @@ git clone $ORIGIN/.
 git checkout $SHA1
 
 
-mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am package
+mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am install
```
Comment (rouazana) [inline dockerfiles/compilation/java-8/integration_tests.sh]: this will break the cache for future build, no?

Comment (mbaechler) [reply dockerfiles/compilation/java-8/integration_tests.sh]: of course, it requires an update of the jenkins side

**Comment by aduprat** [comment]
```

[ERROR] Failures: 
[ERROR]   RabbitMQEventBusTest$LifeCycleTest$SingleEventBus.dispatchShouldWorkAfterRestartForOldKeyRegistration:359
```
test this please

**Comment by trantienduchn** [comment]
```
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6] [INFO] Results:
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6] [INFO] 
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6] [ERROR] Failures: 
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6] [ERROR]   RabbitMQEventBusTest$LifeCycleTest$SingleEventBus.dispatchShouldWorkAfterRestartForOldRegistration:334 
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6] mailboxListener.event(
[fdd2fd444c20221a58cef539bf9ca9583a2c02b6]     org.apache.james.mailbox.events.MailboxListener$MailboxAdded@bf1a1130
```
test this please

**Pull request merged**